packages/swapper/src/swappers/zrx/getZrxMinMax/getZrxMinMax.ts

```diff
@@ -15,7 +15,7 @@ export const getZrxMinMax = async (sellAsset: Asset, buyAsset: Asset): Promise<M
     tokenId: sellAsset.tokenId
   })
 
-  const minimum = bn(1).dividedBy(bnOrZero(usdRate)).toString()
+  const minimum = bn(1).dividedBy(usdRate).toString()
```

Comment:
can't safely divide here - plz use bnorzero

Reply:
Right you are - addressed in https://github.com/shapeshift/lib/pull/639/commits/27b8bebf2fc5d05e2ba07e9212ee26b796aed351